### PR TITLE
Add the reported timeline id to the monitor events table.

### PR DESCRIPTION
**Pull request by DimCitus**

### CODE REVIEW

File: src/monitor/pgautofailover--1.5--1.6.sql [hunk 1]
```diff
@@ -405,6 +454,91 @@ CREATE TRIGGER disable_secondary_check
...
+     limit count
+)
+select * from last_events order by eventtime, eventid;
+$$;
+
+comment on function pgautofailover.last_events(text,int,int)
+        is 'retrieve last COUNT events for given formation and group';
+
+grant execute on function pgautofailover.last_events(text,int,int)
+   to autoctl_node;
```
Comment (JelteF) [inline src/monitor/pgautofailover--1.5--1.6.sql:539]: All these grants are actualy not in `pgautofailover.sql`, I guess it doesn't need access to these functions.

**Review by JelteF — APPROVED** [review]
Apart from the inconsistent grants I think this is good to merge.

File: src/monitor/pgautofailover--1.5--1.6.sql (hunk 1 above)
Comment (DimCitus) [reply src/monitor/pgautofailover--1.5--1.6.sql:539]: Other grants on functions are edited the same (only in the upgrade script), and I came to the theory that the upgrade script is run by the superuser and default grants would not be adequate?

Comment (DimCitus) [reply src/monitor/pgautofailover--1.5--1.6.sql:539]: Actually most functions have the grant execute lines, no reason to bypass this one. Added now.

**Pull request merged**